src/app/hearing-part/components/hearings-filter/hearings-filter.component.scss

```diff
@@ -0,0 +1,12 @@
+.listing-details-rb-group {
+  display: inline-flex;
+  flex-direction: column;
+}
+
+.listing-details-rb {
+  margin: 5px;
```

Comment:
As said earlier, I would like to get someone more experienced to guide us through that topic. @zielinskikamil maybe?